src/lib/utils/application-commands/ApplicationCommandRegistries.ts

```diff
@@ -115,6 +131,21 @@ export async function handleBulkOverwrite(commandStore: CommandStore, applicatio
 	}
 
 	// Handle global commands
+	await retry(() => handleBulkOverwriteGlobalCommands(commandStore, applicationCommands, foundGlobalCommands), bulkOVerwriteRetries);
+
+	// Handle guild commands
+	for (const [guildId, guildCommands] of Object.entries(foundGuildCommands)) {
+		await retry(() => handleBulkOverwriteGuildCommands(commandStore, applicationCommands, guildId, guildCommands), bulkOVerwriteRetries);
+	}
```

Comment:
Given current implementation, these retries are useless. The error is never thrown out from the methods. AbortError should ideally be thrown up, other errors should still go through emitBulkOverwriteError